### PR TITLE
Testing: Upgrade of flake8 version; #4162

### PR DESCRIPTION
**Pull request by bari12**
This will still break until #4564 is merged, since I did not fix the flake8 errors there. Will rebase once merged.

### CODE REVIEW

**Comment by bziemons** [comment]
Hi Martin, could you also upgrade pylint? I need this for compatibility with Python 3.9 and pylint 2.8.2 is compatible with Python 3.6 and works well: 
[Change on Py3.8/3.9 branch](https://github.com/bziemons/rucio/commit/06ef24c494c5027a0182c37ed190e9ad83432ec8#diff-f738270eb643c461e5b33c8a8778325c400c5071554bf0aac10d017600eb2087R9)
[Test run with that change](https://github.com/bziemons/rucio/actions/runs/792427880)

**Comment by bari12** [comment]
Yes, I am already preparing a second PR for this... 😄 
Just wanted to first check if there are any major changes I have to make due to newer pylint. Good to know it is working for you! Thanks

File: lib/rucio/api/subscription.py [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+# - Martin Barisits <martin.barisits@cern.ch>, 2012-2021
+# - Mario Lassnig <mario.lassnig@cern.ch>, 2012-2013
+# - Cedric Serfon <cedric.serfon@cern.ch>, 2013-2017
+# - Thomas Beermann <thomas.beermann@cern.ch>, 2014
+# - Hannes Hansen <hannes.jakob.hansen@cern.ch>, 2018
+# - Andrew Lister <andrew.lister@stfc.ac.uk>, 2019
+# - Eli Chadwick <eli.chadwick@stfc.ac.uk>, 2020
+# - Patrick Austin <patrick.austin@stfc.ac.uk>, 2020
+# - James Perry <j.perry@epcc.ed.ac.uk>, 2020
+
```
Comment (bziemons) [inline lib/rucio/api/subscription.py:27]: old copyright needs to be removed

File: lib/rucio/web/ui/flask/common/utils.py [hunk 2]
```diff
@@ -587,7 +582,7 @@ def oidc_auth(account, issuer, ui_vo=None):
     auth_url = auth.get_auth_oidc(account, vo=ui_vo, **kwargs)
     if not auth_url:
         return render_template("problem.html", msg="It was not possible to get the OIDC authentication url from the Rucio auth server. "
-                                + "In case you provided your account name, make sure it is known to Rucio.")   # NOQA: W503
+                               + "In case you provided your account name, make sure it is known to Rucio.")   # NOQA: W503
```
Comment (bziemons) [inline lib/rucio/web/ui/flask/common/utils.py]: maybe make a proper multiline string out of this:
```py
msg=("ab"
     "cd")
```

File: lib/rucio/api/subscription.py (hunk 1 above)
Comment (bziemons) [reply lib/rucio/api/subscription.py:27]: :ping_pong: 

Hint: it is below this header

**Review by bziemons — CHANGES_REQUESTED** [review]

**Review by bziemons — APPROVED** [review]

**Pull request merged**